condor_watch_q.py

```diff
@@ -484,15 +480,23 @@ def watch_q(
                 )
                 msg += [""]
 
+            terminal_rows, terminal_columns = os.popen("stty size", "r").read().split()
+            terminal_columns = int(terminal_columns)
+
+            for row in range(len(msg)):
```

Comment:
This is clever enough that it probably deserves a comment to explain why it's happening :)